src/tzdata/TZData.jl

```diff
@@ -16,7 +16,7 @@ end
 
 const ARCHIVE_DIR = joinpath(DEPS_DIR, "tzarchive")
 const TZ_SOURCE_DIR = joinpath(DEPS_DIR, "tzsource")
-const COMPILED_DIR = joinpath(DEPS_DIR, "compiled")
+const COMPILED_DIR = joinpath(DEPS_DIR, string("compiled_julia_v", VERSION))
```

Comment:
I'd rather use separate sub-directories:
```suggestion
const COMPILED_DIR = joinpath(DEPS_DIR, "compiled", string(VERSION))
```
This requires that we change [this line](https://github.com/JuliaTime/TimeZones.jl/blob/01d025ae6420251e3e075f2dd1eae3f7a9b2521e/src/tzdata/build.jl#L107) to use `mkpath`

Reply:
I considered that but then i would also have to change that other line